go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/eclipse-kanto/update-manager
 
-go 1.17
```

Comment:
Why is the go version increased?

Reply:
I have used _slices.Contains_ which comes with 1.18.